app/containers/Vaults/constants.js

```diff
@@ -56,11 +56,10 @@ export const AMPLIFY_VAULTS_ADDRESSES = [
 export const ZAP_V2_3CRV = '0x579422a1c774470ca623329c69f27cc3beb935a1';
 
 export const LAZY_APE_ADDRESSES = [
-  '0x2a38B9B0201Ca39B17B460eD2f11e4929559071E',
-  '0x4B5BfD52124784745c1071dcB244C6688d2533d3',
-  '0x8ee57c05741aA9DB947A744E713C15d4d19D8822',
-  '0xD6Ea40597Be05c201845c0bFd2e96A60bACde267',
-  '0x84E13785B5a27879921D6F685f041421C7F482dA',
+  '0x6Ede7F19df5df6EF23bD5B9CeDb651580Bdf56Ca',
+  '0xC4dAf3b5e2A9e93861c3FBDd25f1e943B8D87417',
+  '0x5fA5B62c8AF877CB37031e0a3B2f34A78e3C56A6',
+  '0x3B96d491f067912D18563d56858Ba7d6EC67a6fa ',
```

Comment:
There is a space at the end of the string, so button does not displays for crvUSDN